src/client/requests/intraledger_send_usd.rs

```diff
@@ -0,0 +1,72 @@
+use graphql_client::reqwest::post_graphql;
+use rust_decimal::Decimal;
+
+use crate::{
+    client::{
+        queries::{
+            intra_ledger_usd_payment_send, query_me::WalletCurrency, IntraLedgerUsdPaymentSend,
+            IntraLedgerUsdPaymentSendInput,
+        },
+        GaloyClient,
+    },
+    errors::{api_error::ApiError, CliError},
+};
+
+impl GaloyClient {
+    pub async fn intraleger_send_usd(
+        &self,
+        username: String,
+        amount: Decimal,
+        memo: Option<String>,
+    ) -> Result<(), CliError> {
+        let me = self.me().await?;
+        let usd_wallet_id = me
+            .default_account
+            .wallets
+            .iter()
+            .find(|wallet| wallet.wallet_currency == WalletCurrency::USD)
+            .map(|wallet| &wallet.id)
+            .expect("Can't get USD wallet")
+            .to_owned();
+
+        let recipient_wallet_id = self.default_wallet(username).await?;
+        let input = IntraLedgerUsdPaymentSendInput {
+            amount,
+            memo,
+            recipient_wallet_id,
+            wallet_id: usd_wallet_id,
+        };
+
+        let variables = intra_ledger_usd_payment_send::Variables { input };
+
+        let response_body = post_graphql::<IntraLedgerUsdPaymentSend, _>(
+            &self.graphql_client,
+            &self.api,
+            variables,
+        )
+        .await
+        .map_err(|_| ApiError::IssueGettingResponse)?;
```

Comment:
don't eat the error

Reply:
Addressed in a106ff98bbe1579b58865e8ac62717b0fafca644